keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1234,8 +1258,22 @@ type DeviceAttribute struct {
 
 // DeviceAttributeMaxValueLength is the maximum length of a string or version attribute value.
 const DeviceAttributeMaxValueLength = 64
+
+// DeviceCapacity is a single entry in [BasicDevice.Capacity].
```

Comment:
This type may be used in other device models, so the comment shouldn't refer to the specific field, IMO.

Reply:
Ack.